packages/app/src/components/Root/Root.tsx

```diff
@@ -101,9 +103,24 @@ const getMenuItem = (menuItem: ResolvedMenuItem, isNestedMenuItem = false) => {
   );
 };
 
+const GlobalFABProvider = ({ context }: { context: ScalprumMountPoint[] }) => {
+  return context.map(({ Component }, index) => {
+    return (
+      <ErrorBoundary
+        // eslint-disable-next-line react/no-array-index-key
+        key={index}
+      >
+        <Component />
+      </ErrorBoundary>
+    );
+  });
+};
```

Comment:
Can you extract this into components/Root/ApplicationListener and components/Root/ApplicationProvider?

You can then also have this code in that component:

```tsx
  const { mountPoints } = useContext(DynamicRootContext);
```

This makes it easier to separate/change/test Root and this new components in the future.

That's the same idea I added as feedback to #2168 :)